megalinter/descriptors/kubernetes.megalinter-descriptor.yml

```diff
@@ -53,23 +56,14 @@ linters:
     # Count errors when either file is invalid, or an error occurred (e.g. couldn't find schema for this object)
     cli_lint_errors_count: regex_sum
     cli_lint_errors_regex: "(?:Invalid|Errors): ([0-9]+)"
-    linter_banner_image_url: https://user-images.githubusercontent.com/19731161/142411871-f695e40c-bfa8-43ca-97c0-94c256749732.png
-    linter_text: |
-      `kubeconform` is a schema-aware Kubernetes manifest validation tool, that tends to have more up-to-date schema definitions than `kubeval`.
     examples:
       - kubeconform myfile.yml
       - kubeconform -ignore-missing-schemas -skip SomeCRD,AnotherCRD -kubernetes-version '1.18.0' -strict myfile.yml
     install:
+      apk:
+        - go
       dockerfile:
-        - |
-          RUN ML_THIRD_PARTY_DIR="/third-party/kubeconform" \
-              && KUBECONFORM_VERSION=v0.5.0 \
-              && mkdir -p ${ML_THIRD_PARTY_DIR} \
-              && wget -P ${ML_THIRD_PARTY_DIR} -q https://github.com/yannh/kubeconform/releases/download/$KUBECONFORM_VERSION/kubeconform-linux-amd64.tar.gz \
-              && tar xf ${ML_THIRD_PARTY_DIR}/kubeconform-linux-amd64.tar.gz --directory ${ML_THIRD_PARTY_DIR} \
-              && mv ${ML_THIRD_PARTY_DIR}/kubeconform /usr/local/bin \
-              && rm ${ML_THIRD_PARTY_DIR}/kubeconform-linux-amd64.tar.gz \
-              && find ${ML_THIRD_PARTY_DIR} -type f -not -name 'LICENSE*' -delete -o -type d -empty -delete
+        - RUN go install github.com/yannh/kubeconform/cmd/kubeconform@latest
```

Comment:
@nvuillam does this linter need the go compiler for its runtime? If not, this linter could be installed with a stage like I already changed once for dustilock or revive, by using a stage to install. 

https://github.com/oxsecurity/megalinter/blob/46b3721de31e2a1b5f0416512191f3d2b5f93b1f/megalinter/descriptors/go.megalinter-descriptor.yml#L77-L88

https://github.com/oxsecurity/megalinter/blob/46b3721de31e2a1b5f0416512191f3d2b5f93b1f/megalinter/descriptors/repository.megalinter-descriptor.yml#L123-L130